src/pages/main/Deliveries.tsx

```diff
@@ -16,7 +16,7 @@ export default function Deliveries() {
         buttonText={translate('deliveries.labels.newDelivery')}
         buttonClickHandler={handleClickOpen}
         //For example. Make Deliveries table
-        table={<VendorsTable />}
+        table={<NewDeliveryTable />}
```

Comment:
Component should jsut be called `DeliveriesTable` no need for `New`

Reply:
Fixed